### PR TITLE
Fix UG bugs and add constraints table

### LINKED ISSUE #204: [PE-D][Tester A] Command Summary in UG is missing Exit command
A minor bug, probably a good idea to include exit command within the Command Summary for consistency's sake given that it was mentioned in the User Guide.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: Dominic-Khoo/ped#3

### LINKED ISSUE #208: [PE-D][Tester B] UG: Command summary show the parameter THEME for switch command as optional
**Description**
The command summary table at the end of User Guide show the parameter `THEME` for switch command as optional. However, it is needed for the command to run

**Screenshot**

This may give a user skimming through the command summary the wrong impression (like maybe it automatically toggles?) and lead to him/her trying something like

**Suggested fix**
Remove the square brackets :)

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: SuhailLoya/ped#3

### LINKED ISSUE #215: [PE-D][Tester B] Constraint for tele handle being between 5 and 32 characters not mentioned in UG
**Description**
Constraint for tele handle being between 5 and 32 characters not mentioned in UG. This may lead to unnecessary confusion for users. 

**Screenshot**

**Suggested fix**
Mention this somewhere in UG :)

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: SuhailLoya/ped#5

### PR DESCRIPTION
**Pull request by kimberlytmq**
Fixes #208 (switch should not have square brackets) 
Fixes #204 (missing exit command in command summary)
Fixes #215 (constraint table added for better clarity)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -320,13 +327,32 @@ Furthermore, certain edits can cause the UniLink to behave in unexpected ways (e
...
+| NAME            | - Should only contain alphabets and spaces                                                                                                                                                                                                                                                                                                                                                                                                        | Alice Tan <br> Hubert Koh                              |
+| CONTACT TYPE    | - Should be 'WORK' or 'PERSONAL' <br> - Not case sensitive                                                                                                                                                                                                                                                                                                                                                                                                                           | Work <br> personal                                     |
```
Comment (achl1012) [inline docs/UserGuide.md]: can add the 'SCHOOL' contact type as well

File: docs/UserGuide.md [hunk 2]
```diff
@@ -320,13 +327,32 @@ Furthermore, certain edits can cause the UniLink to behave in unexpected ways (e
...
+| EMAIL           | - Must follow the 'local-part@domain' format <br> <br> **Local Part:** <br> - Can include letters, numbers, underscores (_), periods (.), and dashes (-) <br> - Must not start or end with a special character <br> <br> **Domain:** <br> - Must begin with '@', followed by domain labels separated by periods and dashes <br> - Each domain label must start and end with an alphabet or number <br> - Must end with a domain label of at least 2 characters following a period (e.g. .com, .edu) | ryan_teo@example.com <br> caleb-sim@email.edu.sg       |
+| MODULE          | - Should start with 2 to 4 letters, followed by 4 numbers <br> - May end with optional single letter or a dash and one letter                                                                                                                                                                                                                                                                                                                                                        | CS2103-T <br> HSI1000                                  |
```
Comment (achl1012) [inline docs/UserGuide.md]: I think currently the module name constraints only allow the module name to start with 2-3 letters and not 4 letters

**Review by achl1012 — CHANGES_REQUESTED** [review]
looks good just some minor things to edit!

File: docs/UserGuide.md (hunk 2 above)
Comment (kimberlytmq) [reply docs/UserGuide.md]: I changed it to 4 letters cause GESS1000 mod (it's under another PR)

File: docs/UserGuide.md (hunk 1 above)
Comment (kimberlytmq) [reply docs/UserGuide.md]: updated!

**Review by achl1012 — APPROVED** [review]
lgtm! 😄

**Pull request merged**